### ISSUE TITLE
Release VSTI-Bench benchmark on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [VITA-Group/VLM-3R#1]
Hello,

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as yours got featured: https://huggingface.co/papers/2505.20279.
The paper page lets people discuss about your paper and lets them find artifacts about it (your models for instance),
you can also claim the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

It's great to see the pre-trained models being released on Hugging Face. Would you like to also host the benchmark dataset on Hugging Face?
I see that the Github mentions datasets to be released soon, I wanted to know whether you would be interested in hosting them on the 🤗 hub to improve their discoverability/visibility.
We can add tags so that people find them when filtering https://huggingface.co/datasets.

Would be awesome to make the dataset available on 🤗 , so that people can do:

```python
from datasets import load_dataset

dataset = load_dataset("your-hf-org-or-username/your-dataset")
```
See here for a guide: https://huggingface.co/docs/datasets/loading.

Besides that, there's the [dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer) which allows people to quickly explore the first few rows of the data in the browser.

Let me know if you're interested/need any help regarding this!

Kind regards,

Niels

**Comment by Jian-Zhang-3DV** [comment]
Hi @NielsRogge,

Thank you so much for your interest in our work and for reaching out about hosting VSTI-Bench on the Hugging Face Hub! We really appreciate your suggestion to improve the discoverability and visibility of our benchmark.

## Good News - Already on Hugging Face! 🎉

We're happy to inform you that we've already uploaded the VSTI-Bench dataset to Hugging Face Hub. In fact, we have two datasets available:

### 1. VSTiBench Test Set
- **URL**: https://huggingface.co/datasets/Journey9ni/vstibench
- **Description**: Contains 6,042 test samples for evaluating vision-spatial-temporal intelligence
- **Format**: JSON format with video paths, questions, options, and ground truth answers

### 2. Complete VLM-3R Dataset (including VSTiBench training data)
- **URL**: https://huggingface.co/datasets/Journey9ni/VLM-3R-DATA
- **Description**: Our main instruction tuning dataset that includes training data for both VSiBench and VSTiBench

## Usage

As you suggested, the datasets can be easily loaded using the Hugging Face datasets library:

```python
from datasets import load_dataset

# Load VSTiBench test set
vstibench_test = load_dataset("Journey9ni/vstibench")

# Load complete dataset (including training data)
vlm3r_data = load_dataset("Journey9ni/VLM-3R-DATA")
```

## Dataset Structure

The VSTiBench dataset includes:
- **id**: Unique identifier
- **question**: Spatial-temporal reasoning questions about video content
- **options**: Multiple choice options (when applicable)
- **ground_truth**: Correct answer
- **video_path**: Path to the corresponding video
- **question_type**: Category of the spatial-temporal reasoning task

**Comment by NielsRogge** [comment]
Awesome @Jian-Zhang-3DV !

Could you add a dataset card to both datasets, in which you mention https://huggingface.co/papers/2505.20279? This way, the datasets will show up on the paper page. Note that you can also claim the paper as an author and add the Github URL.

Cheers!
